pkg/modules/observability.go

```diff
@@ -840,21 +837,24 @@ func getIssuerCertServiceObs(op utils.OperatorConfig, obs csmv1.Module, componen
 	// Otherwise, we give them the self-signed cert.
 	if certificate != "" || privateKey != "" {
 		if certificate != "" && privateKey != "" {
-			certificatePath = fmt.Sprintf("%s/moduleconfig/observability/%s", op.ConfigDirectory, CustomCert)
+			certificatePath, err := readConfigFile(obs, cr, op, CustomCert)
+			if err != nil {
+				return yamlString, err
+			}
+
+			yamlString = string(certificatePath)
 		} else {
 			return yamlString, fmt.Errorf("observability install failed -- either cert or privatekey missing for %s custom cert", componentName)
 		}
 	} else {
-		certificatePath = fmt.Sprintf("%s/moduleconfig/observability/%s", op.ConfigDirectory, SelfSignedCert)
-	}
+		certificatePath, err := readConfigFile(obs, cr, op, SelfSignedCert)
```

Comment:
If this is no longer a path but a string or something like that, can we rename the variable to indicate that?

Reply:
Fixed